apps/notifications/models.py

```diff
@@ -339,50 +339,121 @@ class COMMUNITY_MEMBERS_CHOICES(models.TextChoices):
             "All of these",
         )
 
+    class ORGANIZATION_TYPE_CHOICES(models.TextChoices):
+        ARTS_CULTURE = (
+            "Arts and Culture",
+            "Arts and Culture",
+        )
+        BLOCK_CAPTAIN = ("Block Captain",)
+        CIVIC_ENGAGEMENT_ELECTED_OFFICIAL = (
+            "Civic Engagement/Elected Official",
+            "Civic Engagement/Elected Official",
+        )
+        DISABILITIES_ACCESS_FUNCTIONAL_NEEDS = (
+            "Disabilities and Access and Functional Needs",
+            "Disabilities and Access and Functional Needs",
+        )
+        EDUCATION = (
+            "Education (Schools/Colleges/Universities)",
+            "Education (Schools/Colleges/Universities)",
+        )
+        FREE_LIBRARY_OF_PHILADELPHIA = (
+            "Free Library of Philadelphia",
+            "Free Library of Philadelphia",
+        )
+        GENERAL_COMMUNITY_SERVICES = (
+            "General Community Services",
+            "General Community Services",
+        )
+        HEALTHCARE = (
+            "Healthcare",
+            "Healthcare",
+        )
+        IMMIGRANT_REFUGEE_COMMUNITIES = (
+            "Immigrante/Refugee/Communities that speak languages other than English",
```

Comment:
Maybe the client meant "Immigrant"? Since the rest of the sentence & text is in English.

Reply:
Thank you for catching that typo! I'll get it fixed so we can merge and update the client.